statina/crud/find/batches.py

```diff
@@ -0,0 +1,67 @@
+from typing import Iterable, List, Optional, Literal
+
+from pydantic import parse_obj_as
+
+from statina.adapter import StatinaAdapter
+from statina.constants import sort_table
+from statina.crud.utils import paginate
+from statina.models.database import Batch
+
+
+def get_batches_text_query(text: str) -> dict:
+    return {
```

Comment:
like that you made this a separate function. 

What does "$options": "i" do?

Maybe add a doc string that explains